.env.local.example

```
@@ -20,7 +21,7 @@ PASSWORD_HASHERS=django.contrib.auth.hashers.PBKDF2PasswordHasher
 
 GOOGLE_CLIENT_ID=
 GOOGLE_CLIENT_SECRET=
-CORS_ALLOWED_ORIGINS=http://localhost:5173,http://localhost:5174,http://localhost:5175
+CORS_ALLOWED_ORIGINS=http://localhost:5173,http://localhost:5174,http://localhost:5175,http://0.0.0.0:5173
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CORS_ALLOWED_ORIGINS key should go before the GOOGLE_CLIENT_ID key

(UnorderedKey)